include/osquery/system.h

```diff
@@ -289,5 +291,11 @@ Status createPidFile();
 // Notice that they are swapped when compared to POSIX FUNCTION_r.
 struct tm* gmtime_r(time_t* t, struct tm* result);
 struct tm* localtime_r(time_t* t, struct tm* result);
+/**
+ * @brief Windows getter for determining Admin status
+ *
+ * @return A bool indicating if the current process is running as admin
+ */
+bool isUserAdmin();
```

Comment:
Can we get this moved outside of the `WINDOW` compiler gate? And then on POSIX it's just returns `getuid() == 0`?